src/platform/api/controllers/tenant-controller.js

```
@@ -334,6 +394,70 @@ class TenantController {
+   * template stored on the tenant (or the default template) is used.
+   */
+  static async previewPdfTemplate(request, response) {
+    try {
+      const user = request.user;
+      const tenantId = request.params.id;
+      const { templateType, template, pdfBookingLayout } = request.body;
+
+      if (
+        pdfBookingLayout &&
+        !isValidBookingLayout(pdfBookingLayout)
+      ) {
```

ESLint

[error] Replace `⏎········pdfBookingLayout·&&⏎········!isValidBookingLayout(pdfBookingLayout)⏎······` with `pdfBookingLayout·&&·!isValidBookingLayout(pdfBookingLayout)`

(prettier/prettier)